docs/modeling/relationships.rst

```diff
@@ -0,0 +1,236 @@
+.. include:: /rc1-notice.txt
+
+Relationships
+=============
+
+A relationship defines how two entities relate to each other. In a relational database, this is represented by a foreign key constraint.
+
+.. note::
+    Most of the samples in this article use a one-to-many relationship to demonstrate concepts. For examples of one-to-one and many-to-many relationships see the `Other Relationship Patterns`_ section at the end of the article.
+
+.. contents:: In this article:
+    :depth: 3
+
+Definition of Terms
+-------------------
+
+There are a number of terms used to describe relationships
+ - **Dependent entity:** This is the entity that contains the foreign key property(s). Sometimes referred to as the 'child' of the relationship.
+ - **Principal entity:** This is the entity that contains the primary key property(s). Sometimes referred to as the 'parent' of the relationship.
+ - **Foreign key:** The property(s) in the dependent entity that is used to store the values of the principal key property that the entity is related to.
+ - **Principal key:** The property(s) that uniquely identifies the principal entity. This may be the primary key or an alternate key.
+ - **Navigation property:** A property defined on the principal and/or dependent entity that contains a reference(s) to the related entity(s).
+ - **Collection navigation property:** A navigation property that contains references to many related entities.
+ - **Reference navigation property:** A navigation property that holds a reference to a single related entity.
+ - **Inverse navigation property:** When discussing a particular navigation property, this term refers to the navigation property on the other end of the relationship.
+
+The following code listing shows a one-to-many relationship between ``Blog`` and ``Post``
+  - ``Post`` is the dependent entity
+  - ``Blog`` is the principal entity
+  - ``Post.BlogId`` is the foreign key
+  - ``Blog.BlogId`` is the principal key (in this case it is a primary key rather than an alternate key)
+  - ``Post.Blog`` is a reference navigation property
+  - ``Blog.Posts`` is a collection navigation property
+  - ``Post.Blog`` is the inverse navigation property of ``Blog.Posts`` (and vice versa)
+
+.. literalinclude:: configuring/sample/EFModeling.Conventions/Samples/RelationshipFull.cs
+        :language: c#
+        :lines: 12-28
+        :linenos:
+
+Conventions
+-----------
+
+By convention, a relationship will be created when there is a navigation property discovered on a type. A property is considered a navigation property if the type it points to can not be mapped as a scalar type by the current database provider.
+
+.. note::
+    Relationships that are discovered by convention will always target the primary key of the principal entity. To target an alternate key, additional configuration must be performed using the Fluent API.
```

Comment:
E.g. `PersonSSN` would by-convention be the FK to the `Person` navigation property if a property named `PersonId` doesn't exist.